### ISSUE TITLE
TextDecoder.decode() Type Error hangs the whole app

### ISSUE THREAD

**Issue by AndrewBastin** [AndrewBastin/postwoman-extension#10]
**Describe the bug**

Postwomen keeps raising following exception on a failed request, and hanging the app. I have to use browser's task manager to kill it. Screenshot below.

```
TypeError: Failed to execute 'decode' on 'TextDecoder': The provided value is not of type '(ArraryBuffer or ArrayBufferView)
```

**To Reproduce**
1. Send an empty POST requests to `https://www.feishu.cn/approval/openapi/v2/approval/get`

**Expected behavior**

Display the error response without hanging the app.

**Screenshots**

![image](https://user-images.githubusercontent.com/1430856/88879358-a0833900-d25c-11ea-9e75-f0fc9a72441f.png)

**Desktop (please complete the following information):**
 - OS: macOS 10.15.6
 - Browser: Edge 84.0.522.48
 - Postwomen Version: v1.9.7

**Comment by AndrewBastin** [comment]
This issue is from the main Postwoman repo [https://github.com/liyasthomas/postwoman/issues/1032](https://github.com/liyasthomas/postwoman/issues/1032)

**Comment by AndrewBastin** [comment]
Fixed in 9a14e9e017ef530c6ab33d7b1d745405e02b76f8!
